src/marqo/core/models/marqo_query.py

```diff
@@ -59,8 +50,28 @@ class MarqoTensorQuery(MarqoQuery):
 class MarqoLexicalQuery(MarqoQuery):
     or_phrases: List[str]
     and_phrases: List[str]
+
     # TODO - validate at least one of or_phrases and and_phrases is not empty
 
 
 class MarqoHybridQuery(MarqoTensorQuery, MarqoLexicalQuery):
-    pass
+    hybrid_parameters: HybridParameters
+
+    # Core module will use these fields instead of the score_modifiers_lexical and score_modifiers_tensor inside the HybridParameters
+    score_modifiers_lexical: Optional[List[ScoreModifier]] = None
+    score_modifiers_tensor: Optional[List[ScoreModifier]] = None
+    @root_validator(pre=True)
+    def validate_searchable_attributes_and_score_modifiers(cls, values):
```

Comment:
Is it better to have `MarqoHybridQuery` extending `MarqoQuery` and move  `score_modifiers` and `searchable_attributes` down to `MarqoTensorQuery` and `MarqoLexicalQuery`?

Reply:
It uses some of the parent attributes, for example, hybrid query will still use `or_phrases: List[str]`
    `and_phrases: List[str]` when creating its lexical search term.

Reply:
And that is only because of its inheritance from `MarqoLexicalQuery`